lib/galaxy/util/xml_macros.py

```diff
@@ -5,10 +5,12 @@
     List,
 )
 
-from galaxy.util import (
-    Element,
-    parse_xml,
-)
+from galaxy.util import parse_xml
+
+try:
+    from lxml.etree import _Element
+except ImportError:
+    pass
```

Comment:
What error is aabac58533fdb5aa2da7dd679a562aad370c9bb3 fixing exactly? The galaxy-util package doesn't require lxml currently, so mypy should fail??

Reply:
Alright, seeing that again locally:
```
lib/galaxy/util/xml_macros.py:29: error: Variable "galaxy.util.Element" is not valid as a type
[valid-type]
        macros: Dict[str, List[Element]] = {}
                               ^
lib/galaxy/util/xml_macros.py:29: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases
lib/galaxy/util/xml_macros.py:36: error: Element? has no attribute "get"  [attr-defined]
            tokens[m.get("name")] = m.text or ""
                   ^
lib/galaxy/util/xml_macros.py:36: error: Element? has no attribute "text"  [attr-defined]
            tokens[m.get("name")] = m.text or ""
                                    ^
lib/galaxy/util/xml_macros.py:42: error: Element? has no attribute "get"  [attr-defined]
            macro_dict[m.get("name")] = XmlMacroDef(m)
                       ^
Found 4 errors in 1 file (checked 1460 source files)
```

Reply:
Probably some different package versions w.r.t. lib/galaxy/dependencies/pinned-lint-requirements.txt ?

Reply:
absolutely possible, but it also makes sense, doesn't it ? What do you think about https://github.com/galaxyproject/galaxy/pull/14197 ?